java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java

```diff
@@ -102,7 +103,7 @@ String readScriptImpl(String script) {
     try {
       return Resources.toString(url, StandardCharsets.UTF_8);
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      throw new UncheckedIOException(e);
```

Comment:
What is the reasoning behind this change? Could you please explain a bit more? Also in the PR description.

Reply:
Hi @diemol !
Replacing `RuntimeException` by `UncheckedIOException` doesn't explicitly solve any practical problem, it's just a better exception for wrapping `IOException`. 

You Many coding standards recommend to avoid throwing too generic exceptions like `RuntimeException` or `Exception`. And `UncheckedIOException` is exactly for that: it's a "RuntimeException for wrapping IOException".